server/models/Contact.js

```diff
@@ -0,0 +1,62 @@
+const keystone = require('keystone');
+const { Types } = keystone.Field;
+
+const Contact = new keystone.List('Contact');
+
+Contact.add({
+    firstCard: {
+        title: {
+            type: Types.Text,
+            require: true,
+            initial: true,
+            label: "Título do primeiro cartão"
+        },
+        logos: {
+            type: Types.CloudinaryImages,
+            require: true,
+            initial: true,
+            label: "Logos das redes sociais"
+        },
+        links: {
+            type: Types.Url,
+            require: true,
+            initial: true,
+            label: "Links das redes sociais"
+        },
+    },
+    secondCard: {
+        title: {
+            type: Types.Text,
+            require: true,
+            initial: true,
+            label: "Título do segundo cartão"
+        },
+        inputs: {
+            name: {
+                type: Types.Name,
+                require: true,
+                initial: true,
+                label: "Nome do usuário"
+            },
+            phone: {
+                type: Types.Number,
+                require: true,
+                initial: true,
+                label: "Número do usuário"
+            },
+            email: {
+                type: Types.Email,
+                require: true,
+                initial: true,
+                label: "E-mail do usuário"
+            },
+            message: {
+                type: Types.Text,
+                require: true,
+                initial: true,
+                label: "Mensagem do usuário"
+            }
+        }
+    }
+});
```

Comment:
-Remove title from secondCard 
-Rename the inputs items  for "option" or other name
-Remove require and initial from all inputs items
-Change the type from all inputs for "Text"


```js
 secondCard: {
         optionOne: {
              type: Types.Text,
              label: "Nome do Primeiro Campo"
          },
          optionTwo: {
              type: Types.Text,
              label: "Nome do Segundo Campo"
          },
          optionThree: {
              type: Types.Text,
              label: "Nome do Terceiro Campo"
          },
          optionFour: {
              type: Types.Text,
              label: "Nome do Quarto Campo"
          }
}`

```

Optional Change

-Transform the "firstCard" into something static
```js
firstCard: {
          optionOne: {
              name: {
                    type: Types.Text,
                    require: true,
                    initial: true,
                    label: "Nome da Primeira Opção "
              },
              logo: {
                  type: Types.CloudinaryImage,
                  require: true,
                  initial: true,
                  label: "Logos da Primeira Opção"
              },
              link: {
                  type: Types.Url,
                  require: true,
                  initial: true,
                  label: "Link da Primeira Opção"
              }
          },
          optionTwo: {
              name: {
                    type: Types.Text,
                    require: true,
                    initial: true,
                    label: "Nome da Segunda Opção "
              },
              logo: {
                  type: Types.CloudinaryImage,
                  require: true,
                  initial: true,
                  label: "Logos da Segunda Opção"
              },
              link: {
                  type: Types.Url,
                  require: true,
                  initial: true,
                  label: "Link da Segunda Opção"
              }
          },
          optionThree: {
              name: {
                    type: Types.Text,
                    require: true,
                    initial: true,
                    label: "Nome da Terceira Opção "
              },
              logo: {
                  type: Types.CloudinaryImage,
                  require: true,
                  initial: true,
                  label: "Logos da Terceira Opção"
              },
              link: {
                  type: Types.Url,
                  require: true,
                  initial: true,
                  label: "Link da Terceira Opção"
              }
          },
          optionFour: {
              name: {
                    type: Types.Text,
                    require: true,
                    initial: true,
                    label: "Nome da Quarta Opção "
              },
              logo: {
                  type: Types.CloudinaryImage,
                  require: true,
                  initial: true,
                  label: "Logos da Quarta Opção"
              },
              link: {
                  type: Types.Url,
                  require: true,
                  initial: true,
                  label: "Link da Quarta Opção"
              }
          },
}
```

Another possibility, in my opinion, would be transform into something like this

```js
firstCard: {
          optionOne: {
              name: {
                    type: Types.Text,
                    require: true,
                    initial: true,
                    label: "Nome da Primeira Opção "
              },
              logo: {
                  type: Types.CloudinaryImage,
                  require: true,
                  initial: true,
                  label: "Logos da Primeira Opção"
              },
              link: {
                  type: Types.Url,
                  require: true,
                  initial: true,
                  label: "Link da Primeira Opção"
              }
          },
          optionTwo: {
              name: {
                    type: Types.Text,
                    require: true,
                    initial: true,
                    label: "Nome da Segunda Opção "
              },
              logo: {
                  type: Types.CloudinaryImage,
                  require: true,
                  initial: true,
                  label: "Logos da Segunda Opção"
              },
              link: {
                  type: Types.Url,
                  require: true,
                  initial: true,
                  label: "Link da Segunda Opção"
              }
          },
          optionThree: {
              name: {
                    type: Types.Text,
                    require: true,
                    initial: true,
                    label: "Nome da Terceira Opção "
              },
              logo: {
                  type: Types.CloudinaryImage,
                  require: true,
                  initial: true,
                  label: "Logos da Terceira Opção"
              },
              link: {
                  type: Types.Url,
                  require: true,
                  initial: true,
                  label: "Link da Terceira Opção"
              }
          },
          optionFour: {
              name: {
                    type: Types.Text,
                    require: true,
                    initial: true,
                    label: "Nome da Quarta Opção "
              },
              logo: {
                  type: Types.CloudinaryImage,
                  require: true,
                  initial: true,
                  label: "Logos da Quarta Opção"
              },
              link: {
                  type: Types.Url,
                  require: true,
                  initial: true,
                  label: "Link da Quarta Opção"
              }
          },
}
Contact: {
          optionOne: {
                  type: Types.Text,
                  require: true,
                  initial: true,
                  label: "Nome do Primeiro Campo"
          },
          optionTwo: {
                  type: Types.Text,
                  require: true,
                  initial: true,
                  label: "Nome do Segundo Campo"
          },
          optionThree: {
                  type: Types.Text,
                  require: true,
                  initial: true,
                  label: "Nome do Terceiro Campo"
          },
          optionFour: {
                  type: Types.Text,
                  require: true,
                  initial: true,
                  label: "Nome do Quarto Campo"
          }
}
```